lib/opensusebasetest.pm

```diff
@@ -311,6 +311,7 @@ sub wait_boot {
     my $ready_time      = $args{ready_time} // 200;
     my $in_grub         = $args{in_grub} // 0;
     my $nologin         = $args{nologin};
+    my $reallynologin   = $args{reallynologin};
```

Comment:
Is this a joke? I don't get it :D Please find a better name and describe it in the above documentation string